sdk/src/system/Date.ls

```diff
@@ -0,0 +1,38 @@
+/*
+===========================================================================
+Loom SDK
+Copyright 2011, 2012, 2013 
+The Game Engine Company, LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. 
+===========================================================================
+*/
+
+package system {
+    
+/**
+ * Provides native date utility.
+ */
+[Native]
+native class Date {
+    
+    /**
+     * This function wraps the native strftime function from the ctime C++ library.
```

Comment:
This seems to skip a key detail which is that it seems to only output the current time? Is that the expected behavior?